pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.mypy]
+warn_return_any = true
+warn_unused_configs = true
+
+[tool.poetry]
+authors = ["Elixir Cloud AAI <cloud-service@elixir-europe.org>"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.11",
+]
+description = "A short description of the project."
+license = "Apache-2.0"
+maintainers = ["Elixir Cloud AAI <cloud-service@elixir-europe.org>"]
+name = "TusStorageHandler"
+readme = "README.md"
+repository = "https://github.com/elixir-cloud-aai/TusStorageHandler"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+
+[tool.poetry.group.docs.dependencies]
+added-value = ">=0.24.0,<1.0.0"
+docutils = "0.20.1"
+furo = ">=2024.5.6,<2025.0.0"
+m2r2 = "^0.3.3.post2"
+sphinx = "^7.3.7"
+sphinx-autodoc-typehints = "^2.1.0"
+tomli = ">=2.0.1,<3.0.0"
+
+[tool.poetry.group.lint.dependencies]
+ruff = ">=0.4.10,<1.0.0"
+typos = ">=1.22.8,<2.0.0"
+
+[tool.poetry.group.misc.dependencies]
+cruft = ">=2.15.0,<3.0.0"
+jello = ">=1.6.0,<2.0.0"
+pre-commit = ">=3.7.1,<4.0.0"
+
+[tool.poetry.group.security.dependencies]
+bandit = ">=1.7.8,<2.0.0"
+safety = ">=3.2.0,<4.0.0"
+
+[tool.poetry.group.test.dependencies]
+pytest = ">=8.2.2,<9.0.0"
+pytest-cov = ">=5.0.0,<=6.0.0"
+
+[tool.poetry.group.types.dependencies]
+mypy = ">=1.10.0,<2.0.0"
+
+[tool.poetry.scripts]
+TusStorageHandler = "TusStorageHandler.main:main"
```

Comment:
there is no console script in `main.py`, main file doesn't even exist, remove this.